### PR TITLE
Add ability to use custom tags

### PR DESCRIPTION
**Pull request by robert-milan**
This change adds the ability to use custom tags for fakemetrics.

### CODE REVIEW

**Comment by Dieterbe** [comment]
so all metrics get the exact same tags? This seems unrealistic and thus not useful?
alternatively, we could allow to pass html templates to set them dynamically

**Review by Dieterbe — COMMENTED** [review]
a

**Comment by robert-milan** [comment]
I think most tags would be the same with only a few variants such as specific server name or something along those lines.

File: cmd/root.go [hunk 1]
```diff
@@ -100,7 +103,10 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&statsdAddr, "statsd-addr", "", "statsd TCP address. e.g. 'localhost:8125'")
 	rootCmd.PersistentFlags().StringVar(&statsdType, "statsd-type", "standard", "statsd type: standard or datadog")
 
-	rootCmd.PersistentFlags().BoolVarP(&addTags, "add-tags", "t", true, "add tags to generated metrics")
+	rootCmd.PersistentFlags().BoolVarP(&addTags, "add-tags", "t", false, "add tags to generated metrics (default false)")
+	rootCmd.PersistentFlags().IntVar(&numUniqueTags, "num-unique-tags", 0, "a number between 0 and 10. when using add-tags this will make the tags unique (default 0)")
+	rootCmd.PersistentFlags().StringSliceVar(&customTags, "custom-tags", []string{}, "A list of comma separated tags (i.e. \"tag1=value1,tag2=value2\")(default empty)")
+	rootCmd.PersistentFlags().IntVar(&numUniqueCustomTags, "num-unique-custom-tags", 0, "a number between 0 and the length of custom-tags. when using custom-tags this will make the tags unique (default 0)")
```
Comment (Dieterbe) [inline cmd/root.go:109]: this seems rather confusing. how are custom tags "made" unique ?
we should have a chat to see how we can streamline this i think

**Pull request merged**